tox.ini

```diff
@@ -16,6 +16,8 @@ deps =
     astropydev: git+https://github.com/astropy/astropy
     pytest-cov
     pytest-xdist
+    pudb
+    pytest-pudb
```

Comment:
Temporary fix for some issues on my setup, this will probably not be necessary (though I wouldn't mind keeping it in if we officially sanction pudb as the most awestome debugger there is!)